CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * **[Fix]** Fix `double-quoted` warnings in Xcode 12.
 
+### App Center Crashes
+
+* **[Fix]** Fix setting `nil` for attachments in the error report.
```

Comment:
This description is not clear. Add details that it's a listener and that signature of one method is changed.